frontend/package.json

```diff
@@ -10,6 +10,12 @@
     "dev:collaboration-service": "cd ../backend/collaboration-service && npm run dev",
     "dev:matching-service": "cd ../backend/matching-service && npm run dev",
     "dev-all": "concurrently \"npm run dev:frontend\" \"npm run dev:user-service\" \"npm run dev:question-service\" \"npm run dev:matching-service\" \"npm run dev:collaboration-service\"",
+    "setup:frontend": "npm i",
+    "setup:user-service": "cd ../backend/user-service && npm i",
+    "setup:question-service": "cd ../backend/question-service && npm i",
+    "setup:collaboration-service": "cd ../backend/collaboration-service && npm i",
+    "setup:matching-service": "cd ../backend/matching-service && npm i",
+    "setup-all": "concurrently \"npm run setup:frontend\" \"npm run setup:user-service\" \"npm run setup:question-service\" \"npm run setup:matching-service\" \"npm run setup:collaboration-service\"",
```

Comment:
Thanks for creating this tool!!! Was about to create a PR to do that xD
@tlyi @xingjie99 @weidak for your reference.

Reply:
👍🏻